frontend/src/static/js/v_authorship.js

```diff
@@ -245,20 +246,25 @@ window.vAuthorship = {
             ? this.getContributionFromAllAuthors(contributionMap)
             : contributionMap[this.info.author];
 
-        if (lineCnt) {
+        if (this.isValidFile(this.info.isMergeGroup,
+            this.info.author, contributionMap, lineCnt, file.isBinary)) {
```

Comment:
why passing this.info as parameter?

Reply:
Thanks for pointing it out. I will remove it accordingly.